js/build/shortcode-ui.js

```diff
@@ -740,6 +740,24 @@ var editAttributeFieldAttachment = sui.views.editAttributeField.extend( {
 	_openMediaFrame: function(e) {
 		e.preventDefault();
 		this.frame.open();
+		if ( this.model.get( 'value' ) ) {
+			var query_args = {
+				'post__in': [ this.model.get( 'value' ) ],
+				'orderby': 'post__in',
+				'perPage': 1
+			};
+			var attachments = wp.media.query( query_args );
```

Comment:
Not sure. I spent a while digging through the parameters available to `wp.media.query`, but I can't seem to make any of them actually work:

![screen shot 2015-08-13 at 11 04 54 am](https://cloud.githubusercontent.com/assets/665992/9253306/2c5eddbe-41ab-11e5-9923-d19d77d57bda.png)

I'll come back to this later, but I'm not getting it right now...